<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveDlg</class>
 <widget class="QWidget" name="ActiveDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>软件激活</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{color: rgb(255, 255, 255);
background-color: rgb(15, 21, 42);
}
QPushButton{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Mac">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SN">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>机器码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>激活码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Active">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>激活</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
